src/App.js

```diff
@@ -1,23 +1,11 @@
-import logo from './logo.svg';
+import 'bootstrap/dist/css/bootstrap.min.css';
 import './App.css';
+import Nav from './Components/Nav';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Nav />
     </div>
```

Comment:
- [ ] You have done an excellent job with this app, congrats! However, every item is using the same URL:

|1|2|
|---|---|
|<img width="693" alt="Screen Shot 2022-09-09 at 6 05 14 PM" src="https://user-images.githubusercontent.com/83717285/189457220-bb2898ba-3e64-427d-8c90-94f3c37017fe.png">|<img width="687" alt="Screen Shot 2022-09-09 at 6 05 23 PM" src="https://user-images.githubusercontent.com/83717285/189457232-ca928326-8add-437f-833e-30879ec8acb5.png">|

I would suggest using unique URLs for every item e.g: `/Details/1` `Details/2` etc, as this is a requirement for this activity.

<img width="941" alt="Screen Shot 2022-09-09 at 6 07 12 PM" src="https://user-images.githubusercontent.com/83717285/189457504-2b4bfc61-ea4b-4923-9385-2ae4b174c99a.png">

Reply:
Thank you. It has been implemented.